subscription-engine/kafka-ws/Dockerfile

```diff
@@ -36,7 +36,8 @@ RUN apk --no-cache add \
     redis \
     libsasl \
     lz4-libs \
-    tini
+    tini \
+    curl
```

Comment:
If 'curl' is being used for the health check only, we can replace it by 'wget' which is available in alpine images.

HEALTHCHECK --start-period=30s --interval=30s --timeout=10s --retries=3 \
CMD wget http://localhost:9000/health -q -O - > /dev/null 2>&1